File: TwoPointers/IsPalindrome.py
class Solution(object):
    def isPalindrome(self, s):
        start = 0
        end = len(s) - 1
        while(start < end):
            if not s[start].isalnum():
                start += 1
                continue
            if not s[end].isalnum():
                end -= 1
                continue
            if s[start].lower() != s[end].lower():
                print(s[start] + " " + s[end])
                return False
            start += 1
            end -= 1 
        return True

File: TwoPointers/IsPalindrome.cpp
#include <string>

using namespace std;

class Solution {
public:
    bool isPalindrome(string s) {
        string::iterator itS = s.begin();
        string::iterator itE = s.end();
        while(itS < itE)
        {
            if(!isalnum(*itS))
            {
                itS++;
                continue;
            }
            if(!isalnum(*itE))
            {
                itE--;
                continue;
            }
            if(tolower(*itS) != tolower(*itE))
                return false;
            itS++;
            itE--;
        }
        return true;
    }
};